docker-compose.ci.yml

```diff
@@ -1,5 +1,9 @@
 version: '3'
 services:
+  taui:
+    environment:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
```

Comment:
Mastarm uses the AWS Node.js SDK under the hood to do deployments. We have to pass these variables into the deployment container in order to give it credentials.